src/site/content/en/fast/chrome-ux-report/index.md

```diff
@@ -48,7 +48,7 @@ handled for you under the hood. The dashboard shows the distribution of user
 experiences, as captured by key performance metrics, and how it changes over
 time. It also shows how the distributions of qualitative metrics like device
 type and effective connection type change over time. Try the
-[Data Studio Dashboard guide](/chrome-ux-report-data-studio-dashboard).
+[Data Studio Dashboard guide](https://developer.chrome.com/blog/chrome-ux-report-looker-studio-dashboard).
```

Comment:
In addition to the URL change, should we change the page title? (throughout)
```suggestion
[Looker Studio Dashboard guide](https://developer.chrome.com/blog/chrome-ux-report-looker-studio-dashboard).
```

Reply:
Updated.